canonical/snapcraft.io
# Remove hard coded categories

## Done
Replaced hard coded categories with ones from the API

## How to QA
- Go to https://snapcraft-io-4464.demos.haus/beta-store
- Check that there are filters and that they work as expected

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-7509
